panel/src/components/Navigation/OptionsDropdown.vue

```diff
@@ -2,7 +2,6 @@
   <!-- Single option = button -->
   <k-button
     v-if="hasSingleOption"
-    v-bind="options[0]"
```

Comment:
I'm not sure what for? This line causing the `e.apply is not a function` error.

Reply:
The idea is that the first item in the options dropdown will be used to define the toggle button if there's just one option in the dropdown.

Reply:
We're talking about `v-bind`, right? Right now my PR seems to be working great. 

I couldn't foresee where it would error/malfunction. Because we already have `v-if` condition for single and multiple options. Frankly, I have no idea about the solution 🤷‍♂️